### DISCUSSION TITLE
Support for solution components

### DISCUSSION THREAD

**Discussion by TWolverson** [Azure/azure-dev#1218] [Ideas]
I often want to iteratively add components to a solution as it grows. For example, developing integrations with Logic Apps and Service Bus to develop a cloud ESB, I typically want to scaffold a similar Logic App structure every time I expand the solution to handle a new entity. It would be great if I could use azd with a template that is meant to insert a new area inside an existing solution, rather than scaffolding an entirely new solution, with some comprehension of the parent solution's conventions so that they're added in the right place in the source code with appropriate names and so on.

**Comment by savannahostrowski** [comment]
We have an issue tracking this type of scenario: https://github.com/Azure/azure-dev/issues/2386
